src/main/java/com/gpu/RegionGpuPlugin.java

```diff
@@ -180,22 +146,21 @@ public class RegionGpuPlugin extends Plugin implements DrawCallbacks
 	static final String WINDOWS_VERSION_HEADER = "#version 430\n";
 
 	static final Shader PROGRAM = new Shader()
-			.add(GL4.GL_VERTEX_SHADER, "vert.glsl")
-			.add(GL4.GL_GEOMETRY_SHADER, "geom.glsl")
```

Comment:
The gray region shader still uses the geometry shader, so this should also be ported to GL43C